tools/configs/nginx-wisdom.conf

```diff
@@ -1,5 +1,5 @@
 upstream uwsgi {
-    server unix:///var/run/uwsgi/ansible_wisdom.sock;
+    server unix:///var/run/uwsgi/ansible_ai_connect.sock;
```

Comment:
Does [this](https://github.com/ansible/ansible-ai-connect-service/blob/main/tools/configs/uwsgi.ini) need updating too? If so, there's an [equivalent](https://github.com/ansible/ansible-ai-connect-operator/blob/main/roles/model/templates/model.configmap.wisdom-service.yaml.j2) in the Operator code-base that would also need updating.

Reply:
Yes you're right. I will remove this change from the PR.